testing/testcafe/tests/navigation/contextMenu/contextMenu.ts

```diff
@@ -44,5 +44,9 @@ test('Context menu should be shown in the same position when item was added in r
         }, 1000);
       }
     },
-  });
+  }, true, '#contextMenu');
+}).after(async () => {
+  await ClientFunction(() => {
+    (window as any).isItemAdded = 0;
```

Comment:
same here

Reply:
It is typo. I wanted to use delete here
`delete (window as any).isItemAdded`